tests/testapp/settings.py

```diff
@@ -1,5 +1,4 @@
-"""
-Django settings for testapp project.
+"""Django settings for testapp project.
```

Comment:
I personally don't love this.
It doesn't improve readability, rather the contrary.

Docstrings are for humans, not machines. I'd rather have a trailing newline, than a dangling typo.